agenta-backend/agenta_backend/routers/app_router.py

```diff
@@ -86,8 +89,9 @@ async def list_app_variants(
         kwargs: dict = await get_user_and_org_id(stoken_session)
 
         if app_id is not None:
-            access_app = await check_access_to_app(kwargs, app_id=app_id)
-
+            access_app = await check_access_to_app(
+                kwargs, app_id=app_id, check_owner=True
```

Comment:
If we don't, the user using the CLI is going to get a message saying they're not authorized to fetch the app variants. 

🤔 I see it now. Since this endpoint is used by both CLI and Web, I'm thinking of adding an extra param called mode, where "web" is the default and "cli" to enable the check_owner to True? What do you think about this?